CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Change Log
 
+## [0.8.0](https://github.com/redhat-developer/vscode-xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * `Markdown` support for `completion` and `hover` documentation. See [#32](https://github.com/redhat-developer/vscode-xml/issues/32)
+ * Add completion for `comment` and `#region`. See [#54](https://github.com/angelozerr/lsp4xml/issues/54)
```

Comment:
all links to lsp4xml issues should be prefixed with lsp4xml, eg. 
```
[lsp4xml#54](https://github.com/angelozerr/lsp4xml/issues/54)
```

Reply:
@fbricon updated